fog/ledger/server/src/key_image_service.rs

```diff
@@ -185,8 +185,9 @@ impl<E: LedgerEnclaveProxy> KeyImageStoreApi for KeyImageService<E> {
     ) {
         let _timer = SVC_COUNTERS.req(&ctx);
         let tracer = mc_util_telemetry::tracer!();
-        let _guard = mc_util_telemetry::extract_context(&ctx)
-            .with_span(tracer.start("auth"))
+        let tracing_ctx = mc_util_telemetry::extract_context(&ctx);
```

Comment:
In some code below you've used the name `parent_ctx`, I found it a little easier to read when that name was used instead of `tracing_ctx`.

Reply:
updated to `parent_ctx` here and in other locations in [7848847](https://github.com/mobilecoinfoundation/mobilecoin/pull/4091/commits/7848847ecfe00ce9949ab59e1644218860d08f09)